core/src/main/resources/jenkins/security/ApiTokenProperty/help-tokenStore.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    API tokens offer a way to make authenticated CLI or REST API calls.
+    See <a href="https://jenkins.io/redirect/api-token">our wiki</a> for more details.<br/>
+    The username associated with each token is your Jenkins username.<br/>
+    <br/>
+    Some good practices for keeping your API tokens secure are:
+    <ul>
+        <li>Use a different token for each application so that if an application is compromised you can revoke its token individually.</li>
+        <li>Regenerate the tokens every 6 months (depending on your context). We put an indicator concerning the age of the token.</li>
+        <li>Protect it like your password, as it allows other people to access Jenkins as you.</li>
+    </ul>
+</div>
```

Comment:
Does this page also need a comment like "Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, \n\
+  which means that the date may be inaccurate."

As on the "Manage Legacy API Token usage". Otherwise, it may confuse users who see inexplicable values here and don't see the admin page.